tests/test_ptc.py

```diff
@@ -183,7 +186,16 @@ def test_covAstier(self):
         resultsExtract.outputCovariances[-2].rawMeans['C:0,0'] = np.array([np.nan])
         resultsExtract.outputCovariances[-2].rawVars['C:0,0'] = np.array([np.nan])
 
-        resultsSolve = solveTask.run(resultsExtract.outputCovariances,
+        # Force the next-to-last PTC dataset to have a decreased variance to
+        # ensure that the outlier fit rejection works.
+        rawVar = resultsExtract.outputCovariances[-4].rawVars['C:0,0']
```

Comment:
Is this comment incorrect here?  `outputCovariances[-4]` isn't next-to-last, unless I'm confused on the indexing.